routes/User.py

```diff
@@ -0,0 +1,66 @@
+from app import db
+from models.player import Player
+from models.game import Game
+from flask_restful import Resource
+from flask import request
+
+
+class User(Resource):
+    def post(self):
+        body = request.get_json(force=True)
+
+        if "name" in body:
+            new_player = Player(name=body["name"])
+            Player.query.session.add(new_player)
+            Player.query.session.commit()
+            return new_player.to_dict(), 201
+        else:
+            return {"error": "name is required"}, 400
+
+
+    def get(self):
+        player_id = request.args['player_id']
+        player = Player.query.get(player_id)
+
+        if not player:
+            return {"error": f"Could not find player {player_id}"}, 404
+        
+        return {
+            "player": {
+                "id": player.id, 
+                "name": player.name,
+                "uid": player.uid,
+            }
+        }, 200
```

Comment:
Another case where having a method in the `Player.py` file could be useful. 

This also applies to below